Documentation/user-guides/getting-started.md

```diff
@@ -0,0 +1,156 @@
+
+The mission of the Prometheus Operator is to make running Prometheus on top of Kubernetes as easy as possible, while preserving configurability as well as making the configuration Kubernetes native.
+
+To follow this getting started you will need a Kubernetes cluster you have access to. Let's give the Prometheus Operator a spin:
+
+[embedmd]:# (../../deployment.yaml)
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prometheus-operator
+  labels:
+    operator: prometheus
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        operator: prometheus
+    spec:
+      containers:
+       - name: prometheus-operator
+         image: quay.io/coreos/prometheus-operator:v0.6.0
+         resources:
+           requests:
+             cpu: 100m
+             memory: 50Mi
+           limits:
+             cpu: 200m
+             memory: 100Mi
+```
+
+The Prometheus Operator introduces third party resources in Kubernetes to declare the desired state of a Prometheus and Alertmanager cluster as well as the Prometheus configuration. The resources it introduces are:
+
+* [`Prometheus`](../prometheus.md)
+* [`Alertmanager`](../alertmanager.md)
+* [`ServiceMonitor`](../service-monitor.md)
+
+> Important for this guide are the `Prometheus` and `ServiceMonitor` resources. Have a look at the [alerting guide](alerting.md) for more information about the `Alertmanager` resource.
+
+The `Prometheus` resource declaratively describes the desired state of a Prometheus deployment, while a `ServiceMonitor` describes the set of targets to be monitored by Prometheus.
+
+![Prometheus Operator Architecture](images/architecture.png "Prometheus Operator Architecture")
+
+The `Prometheus` resource includes a selection of `ServiceMonitors` to be used, this field is called the `serviceMonitorSelector`.
+
+First, deploy three instances of a simple example application, which listens and exposes metrics on port `8080`.
+
+[embedmd]:# (../../example/user-guides/getting-started/example-app-deployment.yaml)
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: example-app
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: example-app
+    spec:
+      containers:
+      - name: example-app 
+        image: fabxc/instrumented_app
+        ports:
+        - name: web
+          containerPort: 8080
+```
+
+The `ServiceMonitor` has a label selector to select `Service`s and the underlying `Endpoints` objects. The `Service` object for the example application selects the `Pod`s by the `app` label having the `example-app` value. In addition to that the `Service` object specifies the port the metrics are exposed on.
+
+[embedmd]:# (../../example/user-guides/getting-started/example-app-service.yaml)
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: example-app
+  labels:
+    app: example-app
+spec:
+  selector:
+    app: example-app
+  ports:
+  - name: web
+    port: 8080
+```
+
```

Comment:
Can you add the kubectl command to start this? (mostly for consistency with below)